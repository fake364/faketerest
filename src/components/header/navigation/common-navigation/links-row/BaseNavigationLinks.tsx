import React from 'react';
import BaseLink from '../link/BaseLink';
import useTranslation from 'next-translate/useTranslation';

type Props = {};

const BaseNavigationLinks: React.FC<Props> = () => {
	const { t } = useTranslation('main-page');
	return (
		<>
			<BaseLink className="mr-10 text-blue" href={'/about'}>
				{t('links.base.description')}
			</BaseLink>
			<BaseLink className="mr-10" href={'/business'}>
				{t('links.base.business')}
			</BaseLink>
			<BaseLink className="mr-10" href={'/blog'}>
				{t('links.base.blog')}
			</BaseLink>
		</>
	);
};

export default BaseNavigationLinks;
