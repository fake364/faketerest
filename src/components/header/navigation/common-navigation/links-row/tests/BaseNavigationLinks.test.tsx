import React from 'react';
import { shallow } from 'enzyme';
import BaseNavigationLinks from '../BaseNavigationLinks';

describe('BaseNavigationLinks', () => {
  it('should render three BaseLinks', () => {
    const wrapper = shallow(<BaseNavigationLinks />);
    expect(wrapper).toMatchSnapshot();
  });
});
