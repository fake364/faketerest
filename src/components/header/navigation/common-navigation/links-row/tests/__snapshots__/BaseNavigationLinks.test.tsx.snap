// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseNavigationLinks should render three BaseLinks 1`] = `
<Fragment>
  <BaseLink
    className="mr-10 text-blue"
    href="/about"
  >
    links.base.description
  </BaseLink>
  <BaseLink
    className="mr-10"
    href="/business"
  >
    links.base.business
  </BaseLink>
  <BaseLink
    className="mr-10"
    href="/blog"
  >
    links.base.blog
  </BaseLink>
</Fragment>
`;
